tools/warning-to-plist/README.md

```diff
@@ -0,0 +1,126 @@
+# warning-to-plist
+`warning-to-plist` is a python tool which creates .plist files from the given
+output.
+
+## Table of Contents
+* [Install guide](#install-guide)
+* [Usage](#usage)
+* [Sanitizers](#sanitizers)
+  * [Undefined Behaviour Sanitizer](#undefined-behaviour-sanitizer)
+  * [Address Sanitizer](#address-sanitizer)
+  * [Memory Sanitizer](#memory-sanitizer)
+  * [Thread Sanitizer](#thread-sanitizer)
+* [License](#license)
+
+## Install guide
+```sh
+# Create a Python virtualenv and set it as your environment.
+make venv
+source $PWD/venv/bin/activate
+
+# Build and install warning-to-plist package.
+make package
+```
+
+## Usage
+```sh
+usage: warn-to-plist [-h] -o OUTPUT_DIR -t TYPE [TYPE ...] [-c] [-v] [file]
+
+Creates plist files from the given output.
+
+positional arguments:
+  file                  Output result file which will be parsed and used to
+                        generate plist files. If it is not given standard
+                        input will be used as the input.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -o OUTPUT_DIR, --output OUTPUT_DIR
+                        Generate .plist output files in the given folder.
+  -t TYPE [TYPE ...], --type TYPE [TYPE ...]
+                        Specify the format of the warning output. Currently
+                        supported output types are: asan, tsan, ubsan, msan,
+                        clang-tidy.
```

Comment:
I agree that the user should use CodeChecker on Clang Tidy instead of this, but if we change the type from  `clang-tidy` to a general type (`diag-msg`) we will loose information that the output was generated by Clang Tidy. We use this information when we print out the plist: https://github.com/Ericsson/codechecker/blob/3d525ae4e7432cfa4fbca628c1875e7a3021aa17/tools/warning-to-plist/warning_to_plist/converter/plist_converter.py#L124